### PR TITLE
Improve CD process

### LINKED ISSUE #828: Add support for spark-operator
**Describe the issue**

Use spark-operator to replace finkctl and local fink install

**Expected behaviour**

Improve gitops support for fink-broker

### PR DESCRIPTION
**Pull request by fjammes**
- increase argoCD usage
- use Spark operator
- use Minio operator
- add Helm chart for fink-broker
- Improve logging management
- Use finkctl to create kafka secret
- Bump ciux to v0.0.3-rc4
- Bump ktbx to v1.1.3-rc1
- Increase sync checks in CI
  * Wait for input topic to exist
  * Wait for fink-producer secret to appear

Linked to issue(s): Closes #828 

## How was this patch tested?

`noscience` CI is passing, `science` CI is broken.

### CODE REVIEW

**Comment by fjammes** [comment]
CI is passing on GHA. Did not test the science module as CI is broken...

File: bin/raw2science.py [hunk 1]
```diff
@@ -84,6 +84,23 @@ def main():
             latestfirst=False,
         )
 
+    # DEBUG STATEMENTS
+    logger.info("XXXXXXXXXXXXX test logger")
```
Comment (JulienPeloton) [inline bin/raw2science.py:88]: I think you can remove these debug statements.

File: bin/stream2raw.py [hunk 2]
```diff
@@ -109,11 +114,13 @@ def main():
         spark.stop()
 
     # Flatten the data columns to match the incoming alert data schema
+    logger.debug("Flatten the data columns to match the incoming alert data schema")
     cnames = df_decoded.columns
     cnames[cnames.index("decoded")] = "decoded.*"
     df_decoded = df_decoded.selectExpr(cnames)
 
     # Partition the data hourly
+    logger.debug("Partition the data hourly")
```
Comment (JulienPeloton) [inline bin/stream2raw.py:123]: Actually the comment is wrong now -- we do not partition anymore here. So you can remove these two lines.

**Review by JulienPeloton — APPROVED** [review]
Thanks @fjammes -- it looks good to me. I left two comments, but then it can be merged.

File: bin/raw2science.py (hunk 1 above)
Comment (fjammes) [reply bin/raw2science.py:88]: :+1:

File: bin/stream2raw.py (hunk 2 above)
Comment (fjammes) [reply bin/stream2raw.py:124]: :+1:

**Pull request merged**